service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocumentUploadedNotificationService.java

```diff
@@ -58,16 +60,24 @@ public Set<String> getSecondaryLocalAuthorityRecipients(CaseData caseData) {
         return localAuthorityRecipients.getRecipients(recipientsRequest);
     }
 
-    public Set<String> getRepresentativeEmails(CaseData caseData, DocumentUploaderType userType) {
+    public Set<String> getRepresentativeEmails(CaseData caseData) {
         List<RepresentativeRole.Type> roles = List.of(CAFCASS, RESPONDENT);
         HashSet<String> emails = representativesInbox.getRepresentativeEmailsFilteredByRole(caseData,
             DIGITAL_SERVICE, roles);
         emails.addAll(representativesInbox.getRespondentSolicitorEmails(caseData, DIGITAL_SERVICE));
+        emails.addAll(representativesInbox.getChildrenSolicitorEmails(caseData, DIGITAL_SERVICE));
+        return emails;
+    }
 
-        if (userType == SOLICITOR) {
-            emails.addAll(representativesInbox.getChildrenSolicitorEmails(caseData, DIGITAL_SERVICE));
+    public void sendNotificationForCourtBundleUploaded(CaseData caseData, Set<String> recipients,
+                                                       String hearingDetails) {
```

Comment:
Does this need a separate function because it's a different email template for court bundles?